app/Login.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+const path = require('path');
+const globals = require('../server/globals.js');
+const request = require('superagent');
+const render = require('react-dom');
+
+import { browserHistory } from 'react-router';
+
+class LoginScreen extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div id="LoginScreen">
+        <div id="loginForm">
+          <h1>BUDGHETTO</h1>
+          <form form action="" onSubmit={this.handleSubmit}>
+            <label for="username">
+              Username
+            </label>
+            <input value="tiivi.taavi@budghetto.space" type="text" id="username"></input>
```

Comment:
Warning: Failed form propType: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`. Check the render method of `LoginScreen`.